### DISCUSSION TITLE
Git repository  url cant find it

### DISCUSSION THREAD

**Discussion by lucad1971** [nvaccess/nvda#14201] [General]
sorry for the dumb question but i am new to github not to programming..
where do i find the git url to download source code whit git?
i click on the code  link of a project, but then  no url appears on the page to use in git.
Thanks.

**Comment by rperez030** [comment]
In the case of NVDA, the URL is part of the readme. In any Github project, under the code section, somewhere below the branch name, there is a button also name code which is in a collapsed state. By expanding this you will find the different URLs to clone the repository with the git clone command. In the case of NVDA, it is convenient to use git -recursive so that you get all the submodules which you will need to build the sources.



From: lucad1971 ***@***.***>
Sent: Friday, September 30, 2022 10:28 AM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nvaccess/nvda] Git url cant find it (Discussion #14201)


sorry for the dumb question but i am new to github not to programming..
where do i find the git url to download source code whit git?
i click on the code link of a project, but then no url appears on the page to use in git.
Thanks.

-
